ccd-definition/CaseEventToComplexTypes/enterApplicantDetailsLa/applicantContact.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "EnterApplicantLa",
+    "CaseEventID": "enterApplicantDetailsLA",
+    "CaseFieldID": "applicantContact",
+    "ListElementCode": "fullName",
```

Comment:
Wonder if we could split this out into first name/last name? Asked service for the applicant is respondent flow and they said it was fine (think Alistair's doing it this way now on his PR).